meilix-artwork/debian/control

```diff
@@ -2,8 +2,8 @@ Source: meilix-artwork
 Section: x11
 Priority: optional
 Maintainer: Mario Behling <mb@mariobehling.de>
-Build-Depends: debhelper (>= 8.0.0), python(>=2.6)
-Standards-Version: 4.1.3
+Build-Depends: debhelper (>= 11), python(>=2.6)
```

Comment:
In the travis build,
`dpkg-checkbuilddeps: Unmet build dependencies: debhelper (>= 11)
dpkg-buildpackage: warning: build dependencies/conflicts unsatisfied; aborting
dpkg-buildpackage: warning: (Use -d flag to override.)
debuild: fatal error at line 1364:
dpkg-buildpackage -rfakeroot -D -us -uc failed`
Please check for travis in local machine
Weirdly the build is passing

Reply:
well , what i am thinking it is for signing thing error , as you tested and its passing locally on your machine too

Reply:
No, You can see it says `unmet build dependencies` travis is not  same as the local machine
Things now to do will be deploying your application to the travis enviroment and see if you get an error there
If you find a solution then you can update the pr